independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanGenerator.java

```diff
@@ -2228,9 +2229,10 @@ private String getProxyTypeName(BeanInfo bean, String baseName) {
         return proxyTypeName.toString();
     }
 
-    private ResultHandle wrapCurrentInjectionPoint(BeanInfo bean,
-            MethodCreator constructor, InjectionPointInfo injectionPoint, int paramIdx, ResultHandle tccl,
-            ReflectionRegistration reflectionRegistration) {
+    public static ResultHandle wrapCurrentInjectionPoint(BeanInfo bean, MethodCreator constructor,
```

Comment:
Just nitpicking, but I prefer package-private modifier where possible...

Reply:
Hmm, I wonder why I made this public. There's been no need for that... Let me fix.